### PR TITLE
[dv/alert_handler] Sign off alert_handler to V2S

### PR DESCRIPTION
**Pull request by cindychip**
This PR updates the checklist to signoff alert_handler to V2S.

Signed-off-by: Cindy Chen <chencindy@opentitan.org>

### CODE REVIEW

File: hw/ip_templates/alert_handler/doc/checklist.md [hunk 1]
```diff
@@ -131,21 +131,21 @@ Testbench     | [TB_TOP_CREATED][]                    | Done        |
 Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Done        |
 Testbench     | [SIM_TB_ENV_CREATED][]                | Done        |
 Testbench     | [SIM_RAL_MODEL_GEN_AUTOMATED][]       | Done        |
-Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | waived      | Revisit later. Tool setup in progress.
+Testbench     | [CSR_CHECK_GEN_AUTOMATED][]           | Done        |
 Testbench     | [TB_GEN_AUTOMATED][]                  | N/A         |
 Tests         | [SIM_SMOKE_TEST_PASSING][]            | Done        |
 Tests         | [SIM_CSR_MEM_TEST_SUITE_PASSING][]    | Done        |
-Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | waived      | Revisit in V2S.
+Tests         | [FPV_MAIN_ASSERTIONS_PROVEN][]        | N/A         |
```
Comment (msfschaffner) [inline hw/ip_templates/alert_handler/doc/checklist.md:138]: does that mean we are only signing off based on simulation coverage, and do not apply this to any of the FPV testbenches?

**Review by msfschaffner — APPROVED** [review]
LGTM with one question.

File: hw/ip_templates/alert_handler/doc/checklist.md (hunk 1 above)
Comment (cindychip) [reply hw/ip_templates/alert_handler/doc/checklist.md:138]: Yes. That is what I believed.
In V2 we excluded prim_alert_sender and prim_esc_receiver from the coverage.
They are covered in FPV and DV (non-VCS) testbenchs. And they all passed the V2 requirements.
I think here the `FPV` testbenchs we are referring to the sec_cm bench (where we black box cm related modules and check if design will trigger internal alerts).

If what I understand correctly, we will do `N/A` here and mark the `V2S` fpv section to done.

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/ip_templates/alert_handler/doc/checklist.md (hunk 1 above)
Comment (msfschaffner) [reply hw/ip_templates/alert_handler/doc/checklist.md:138]: Ok that SGTM.

**Comment by cindychip** [comment]
Latest regression result:
### [Coverage Dashboard](/usr/local/google/home/chencindy/opentitan/scratch/master/alert_handler-sim-vcs/cov_report/dashboard.html)

|  SCORE  |  LINE   |  COND   |  TOGGLE  |   FSM   |  BRANCH  |  ASSERT  |  GROUP  |
|:-------:|:-------:|:-------:|:--------:|:-------:|:--------:|:--------:|:-------:|
| 96.76 % | 99.88 % | 97.27 % | 99.90 %  | 95.77 % | 99.05 %  | 90.77 %  | 94.65 % |

**Pull request merged**